ui/src/components/mobileservices/MobileServiceView.js

```diff
@@ -1,46 +1,71 @@
+                  });
+                } else {
+                  unboundServices.push({
+                    serviceLogoUrl: serviceIcon,
+                    serviceIconClass: serviceIconClass,
+                    serviceName: serviceName,
+                    serviceInstanceName: instance.serviceInstance.metadata.name,
+                    serviceId: serviceName,
+                    bindingSchema : instance.servicePlan.spec.serviceBindingCreateParameterSchema,
+                    form : instance.servicePlan.spec.externalMetadata.schemas.service_binding.create.openshift_form_definition,
+                    serviceDescription: instance.serviceClass.spec.description,
+                    serviceClassExternalName: instance.serviceClass.spec.externalMetadata.serviceName,
+                    setupText: 'Mobile Metrics SDK setups',
+                  });
+                }
+                
+                this.state.boundServices = boundServices;
+                this.state.unboundServices = unboundServices;
+                this.setState(this.state);
+            })
+          }
+        );
+      
     this.state = {
-      boundServices: [{
-        serviceLogoUrl: 'https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png',
-        serviceName: 'Identity Management',
-        serviceId: 'dh-keycloak-apb-h7k9j',
-        serviceDescription: 'Identity Management - Identity and Access Management',
-        documentationUrl: 'http://docs.keycloak.org',
-        setupText: 'Identity Management SDK setup',
-      }, {
-        serviceLogoUrl: 'https://avatars1.githubusercontent.com/u/3380462?s=200&v=4',
-        serviceName: 'Mobile Metrics',
-        serviceId: 'dh-metrics-apb-wqm5c',
-        serviceDescription: 'Installs a metrics service based on Prometheus and Grafana',
-        setupText: 'Mobile Metrics SDK setups',
-        configuration: [{ key: 'A property', value: 'A Value' }, { key: 'Another property', value: 'Another Value' }],
-      }],
-      unboundServices: [{
-        serviceLogoUrl: 'https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png',
-        serviceName: 'Identity Management',
-        serviceId: 'dh-keycloak-apb-h7k9j',
-        serviceDescription: 'Identity Management - Identity and Access Management',
-        documentationUrl: 'http://docs.keycloak.org',
-        setupText: 'Identity Management SDK setup',
-      }, {
-        serviceLogoUrl: 'https://avatars1.githubusercontent.com/u/3380462?s=200&v=4',
-        serviceName: 'Mobile Metrics',
-        serviceId: 'dh-metrics-apb-wqm5c',
-        serviceDescription: 'Installs a metrics service based on Prometheus and Grafana',
-        setupText: 'Mobile Metrics SDK setups',
-        configuration: [{ key: 'A property', value: 'A Value' }, { key: 'Another property', value: 'Another Value' }],
-      }],
+      boundServices: [],
+      unboundServices: [],
```

Comment:
this is where using Redux will help. You don't need to manually update the state in this component.